test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -489,27 +501,29 @@ test
     )).eql(3);
 
     await challengeEmailPageObject.verifyFactor('credentials.passcode', 'xyz');
-    await challengeEmailPageObject.clickNextButton();
+    await challengeEmailPageObject.clickNextButton('Verify');
     await challengeEmailPageObject.waitForErrorBox();
-    await t.expect(challengeEmailPageObject.getInvalidOTPFieldError()).contains('Invalid code. Try again.');
+    // TODO - This error doesn't show up in v3
+    // await t.expect(challengeEmailPageObject.getInvalidOTPFieldError()).contains('Invalid code. Try again.');
```

Comment:
Hmm, I thinkthis error should show up in v3, I wonder if it isnt being propagated to the field as expected (which would have more to do with the mock file). Can you mark this test as disabled for v3 and create a ticket for us to look into this? I think we want to make sure we match parity as much as possible and disabling this will also impact v2.

Reply:
I would also uncomment this to make sure we are still checking this for v2.